lib/modules/selectedEntry.js

```diff
@@ -144,6 +144,11 @@ export function select(thing, options) {
 	_select(thing, options);
 }
 
+export const selectClosestVisible = _.debounce(() => {
```

Comment:
`_.debounce()` with no delay is a basically a no-op, so you can remove that and `export function selectClosestVisible() {...`

We don't need a debounce for this usage, so @larsjohnsen can add it back in his PR if necessary